error[E0999]: assertion might fail: possible out-of-bounds access
   --> /Users/Exelica/Downloads/project/flux-verify-std/library/core/src/num/int_sqrt.rs:146:22
    |
146 |         let (s, r) = U8_ISQRT_WITH_REMAINDER[n as usize];
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
267 |     let (s, r) = first_stage!(16, n);
    |                  ------------------- in this macro invocation
    |
    = note: this error originates in the macro `first_stage` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0999]: assertion might fail: possible out-of-bounds access
   --> /Users/Exelica/Downloads/project/flux-verify-std/library/core/src/num/int_sqrt.rs:146:22
    |
146 |         let (s, r) = U8_ISQRT_WITH_REMAINDER[n as usize];
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
279 |     let (s, r) = first_stage!(32, n);
    |                  ------------------- in this macro invocation
    |
    = note: this error originates in the macro `first_stage` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0999]: assertion might fail: possible out-of-bounds access
   --> /Users/Exelica/Downloads/project/flux-verify-std/library/core/src/num/int_sqrt.rs:146:22
    |
146 |         let (s, r) = U8_ISQRT_WITH_REMAINDER[n as usize];
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
292 |     let (s, r) = first_stage!(64, n);
    |                  ------------------- in this macro invocation
    |
    = note: this error originates in the macro `first_stage` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0999]: assertion might fail: possible out-of-bounds access
   --> /Users/Exelica/Downloads/project/flux-verify-std/library/core/src/num/int_sqrt.rs:146:22
    |
146 |         let (s, r) = U8_ISQRT_WITH_REMAINDER[n as usize];
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
306 |     let (s, r) = first_stage!(128, n);
    |                  -------------------- in this macro invocation
    |
    = note: this error originates in the macro `first_stage` (in Nightly builds, run with -Z macro-backtrace for more info)


thread 'rustc' panicked at compiler/rustc_trait_selection/src/traits/select/mod.rs:305:24:
`TraitPredicate(<{closure@/Users/Exelica/Downloads/project/flux-verify-std/library/core/src/iter/adapters/copied.rs:30:5: 30:21} as ops::function::FnOnce<(Acc, &T)>>, polarity:Positive)` has escaping bound vars, so it cannot be wrapped in a dummy binder.
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
note: uncaught panic [def_id: DefId(0:6128 ~ core[8058]::iter::adapters::copied::copy_fold), span: /Users/Exelica/Downloads/project/flux-verify-std/library/core/src/iter/adapters/copied.rs:29:1: 29:91 (#0)]

error: could not compile `core` (lib) due to 4 previous errors
